### DISCUSSION TITLE
Migrating from secondary drive

### DISCUSSION THREAD

**Discussion by DailenG** [belowaverage-org/SuperGrate#98] [Q&A] [answered]
Botched Windows Update rendered a machine unable to boot thanks to some driver issues and i opted to just put in a blank drive and copy his stuff over but couldn't figure out how to migrate a profile from D:\ drive, perhaps not feasible with offline Windows?

**Comment by krisdb2009** [comment, marked as answer]
I think it is possible with stand-alone USMT, but Super Grate is not aware of this scenario therefore not possible with it.

**Comment by Deng-Xian-Sheng** [comment]
I am doing this and have already achieved some significant results
I have studied USMT, but it is actually easy to understand. I have not found any good articles on USMT online, and the Microsoft documentation is difficult and obscure

---
Although I am stuck with the problem now, I have still achieved some success

A few hours later, after I finish work, I will write a brief Chinese article explaining the key parts of learning USMT, which have never been found online and are the most crucial. This is what I have gained through experience (energy and time)

I will also push the code and XML files that are already being written to the GitHub repository, you can take a look

I am committed to transferring applications on offline systems

**Comment by Deng-Xian-Sheng** [comment]
This is the link to the article I posted. It may still be under review, but the review process is fast
https://blog.csdn.net/Deng_Xian_Sheng/article/details/139780203?csdn_share_tail=%7B%22type%22%3A%22blog%22%2C%22rType%22%3A%22article%22%2C%22rId%22%3A%22139780203%22%2C%22source%22%3A%22Deng_Xian_Sheng%22%7D

**Comment by Deng-Xian-Sheng** [comment]
This is the warehouse link, writing the article has taken up too much of my time. I am rushing home and will update the code later

https://github.com/Deng-Xian-Sheng/USMT-Helper

**Comment by Deng-Xian-Sheng** [comment]
To prevent misunderstandings, I will provide some non-technical explanations

It is a good way to solve the problem. There are too few people using USMT. The discussion in this warehouse is one of the few discussions about USMT on the Internet

I have no ideas that motivate the author to make changes or compete. I want to solve my and people's problems, make the world a better place, and make an attempt

The author is great, and being an open-source person is very cool
